src/files.rs

```diff
@@ -19,16 +19,32 @@ pub fn read_file(path: &Path) -> Fallible<Vec<u8>> {
     std::fs::read(&path).map_err(|err| format!("{}: {}", err, path.display()).into())
 }
 
+fn files_differ(old: &Path, new: &Path) -> Fallible<bool> {
+    if !(old.is_file() && new.is_file()) {
```

Comment:
As below, turning I/O errors into changes should suffice.

Reply:
You may have had a less verbose implementation in mind?